### ISSUE TITLE
allow manual entry of location

### ISSUE THREAD

**Issue by Etheral** [benkaiser/cheap-coke#1]
tried to use this on my desktop but I can't give my location since there's no GPS unit.
would be nice to be able to enter a postcode or pick a store?

**Comment by benkaiser** [comment]
Agreed. For now it uses geo-ip. I'm curious was it able to find your closest store with geo-ip?

**Comment by Etheral** [comment]
sorta... I'm in the north Sydney area and it gave me Marrickville (inner westish).
admittedly just being in Sydney is probably fine given the price in a state doesn't change usually.
